released/packages/coq-approx-models/coq-approx-models.1.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "damien.pous@ens-lyon.fr"
+version: "1.0"
+
+homepage: "https://gitlab.inria.fr/amahboub/approx-models"
+dev-repo: "git+https://gitlab.inria.fr/amahboub/approx-models.git"
+bug-reports: "https://gitlab.inria.fr/amahboub/approx-models/issues"
+license: "CECILL-B"
+
+synopsis: "Rigorous approximations with a posteriori verified operations"
+description: """
+This is a Coq library to verify rigorous approximations of univariate
+functions on real numbers. Based on interval arithmetic, this library
+also implements a technique of validation a posteriori based on the
+Banach fixed-point theorem. We moreover provide an implementation of
+verified Chebyshev approximations."""
+
+build: [make "-j%{jobs}%" ]
+install: [make "install"]
+depends: [
+  "coq" {(>= "8.13" & < "8.14~") | (= "dev")}
+  "coq-interval"
+  "coq-coquelicot" {(= "3.2.0")}
+]
+
+tags: [
+  "category:Computer Science/Approximation Theory"
```

Comment:
This does not look like a `Computer Science` category. Would it not be better to put under `Mathematics`?

Capitalization is allowed in keywords, so you might just as well use `"keyword:Chebyshev polynomials"`.

Also, please add a tag `"date:YYYY-MM-DD"`.

Reply:
ok, will do that now